integration/simulation/transaction_injector.go

```diff
@@ -140,6 +151,26 @@ func (m *TransactionInjector) Start() {
 	m.fullyStoppedChan <- true
 }
 
+func (m *TransactionInjector) deployERC20(w wallet.Wallet) {
+	// deploy the ERC20
```

Comment:
Can a comment be added, or method name changed to make sure it's not confused with the l1 contract deployment